RELEASE-NOTES.txt

```diff
@@ -1,3 +1,8 @@
+1.15.0
+------
+* Fix issue when multiple media selection adds only one image or video block
+* Fix issue when force Touch app shortcut doesn't work properly selecting "New Photo Post"
```

Comment:
Should we specify the platform?
The first one is Android only and the second one is iOS only.